index.html

```diff
@@ -11,13 +11,13 @@
     />
 
```

Comment:
Based on project requirements, in pull request you had to put links to corresponding changes:
![image](https://user-images.githubusercontent.com/77038610/145044612-c5e86bfd-e729-413a-ad83-74dd935480d5.png)

 Please add the links in your pull request summary. You can see this [pull request summary](https://github.com/orozCoding/myportfolio/pull/8) to get an idea what i am talking about :blush: